backend/src/main/kotlin/blue/mild/covid/vaxx/service/IsinService.kt

```diff
@@ -205,7 +215,7 @@ class IsinService(
             logger.info("No vaccine expiration provided for vaccination ${vaccination.vaccinationId}. Skipping vaccination creating in ISIN.")
             return false
         }
-        if (patient.isinReady != true) {
+        if (vaccination.doseNumber == 1 && patient.isinReady != true) {
             logger.info("Patient ${patient.id} is not ISIN ready. Skipping vaccination creating in ISIN.")
```

Comment:
tady možná lepší popis toho co se loguje, nevím jestli i nerozdělit

Reply:
původně jsem měl první a druhou dávku rozdělenou, ale byl tam obrovský overlap v kódu, takže jsem to dal do jedné funkce